src/Mvc/Mvc.Analyzers/src/Startup/StartupAnalyzer.Events.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.AspNetCore.Analyzers
+{
+    public partial class StartupAnalzyer : DiagnosticAnalyzer
```

Comment:
`internal` to keep with the theme?

Reply:
If that's allowed sure.

Reply:
Other analyzers in this project are internal. Leaving public for consistency.